podium/talks/views.py

```diff
@@ -1,6 +1,9 @@
 from django.shortcuts import render, redirect
-
+from django.http import HttpResponseRedirect
```

Comment:
Please do not submit commits with only import additions.  Import additions should be included with the commits that use those imports so that they are clearly linked.